src/js/pensions/components/IntroductionPage.jsx

```diff
@@ -19,19 +19,19 @@ class IntroductionPage extends React.Component {
       <div className="schemaform-intro">
         <FormTitle title="Apply for Veterans pension benefits"/>
         <p>Equal to VA Form 21P-527EZ (Application for Pension).</p>
-        {/* <SaveInProgressIntro */}
-        {/*   prefillEnabled={this.props.route.formConfig.prefillEnabled} */}
-        {/*   pageList={this.props.route.pageList} */}
-        {/*   startText="Start the Pension Application" */}
-        {/*   {...this.props.saveInProgressActions} */}
-        {/*   {...this.props.saveInProgress}> */}
-        {/*   Please complete the 21-527EZ form to apply for pension benefits. */}
-        {/* </SaveInProgressIntro> */}
-        <div className="usa-alert usa-alert-warning">
-          <div className="usa-alert-body">
-            We’re sorry. Our online application isn’t available right now. We’re working to make it easier for you to apply for benefits online, so please check back later to see our new and improved process.
-          </div>
-        </div>
+        <SaveInProgressIntro
```

Comment:
I'm guessing I'm just unfamiliar with the impetus behind this change, but does it make sense to have in this PR?

Reply:
It does not and I thought I removed it!